### PR TITLE
VSS config + mutiny-wasm 0.4.1

### LINKED ISSUE #323: LNURL auth button doesn't work
We changed the APIs around recently. Probably broke that.

### PR DESCRIPTION
**Pull request by benthecarman**
will be needed for next release

Closes #323

### CODE REVIEW

**Comment by futurepaul** [comment]
is there a url we can put in the .env.example?

**Comment by TonyGiorgio** [comment]
This text can be removed: `Beta warning: you can currently only restore on-chain funds. Lightning backup restore is coming soon.`

great fucking work, testing it now.

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
         "@kobalte/core": "^0.9.8",
         "@kobalte/tailwindcss": "^0.5.0",
         "@modular-forms/solid": "^0.13.2",
-        "@mutinywallet/mutiny-wasm": "0.4.0",
+        "@mutinywallet/mutiny-wasm": "0.4.1",
```
Comment (TonyGiorgio) [inline package.json:41]: version the package.json at the top as well

**Review by TonyGiorgio — CHANGES_REQUESTED** [review]
nit, otherwise gtg

**Review by TonyGiorgio — APPROVED** [review]

**Pull request merged**